### DISCUSSION TITLE
Allow placing down paths in a similar way to scenery

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#19109] [Feature requests] [answered]
There's two path modes of path building, one is drawing and the other is building similar to a ride. The drawing works decently similar to scenery, but you cant hold shift to go up or hold ctrl to draw on the same height. I do think this could be a very useful feature to speed up the process of building something like boardwalk/pier or whatnot.

**Comment by ocalhoun6** [comment, marked as answer]
Also see #17607

If any progress is made on this, it might also be a good time to implement #17608 as well.

**Reply by karst** [reply]
Ah yea it's a duplicate of that one, my apologies.
